tutorials/resources.md

```diff
@@ -61,7 +61,7 @@ Ignition will look for GUI plugins on the following paths, in order:
 1. All paths set on the `IGN_GUI_PLUGIN_PATH` environment variable
 2. [GUI plugins that are installed with Ignition Gazebo](https://github.com/ignitionrobotics/ign-gazebo/tree/main/src/gui/plugins)
 3. Other paths added by calling `ignition::gui::App()->AddPluginPath`
-4. `~/.ignition/gui/plugins`
+4. `~/.ignition/gui/<#>/plugins`
```

Comment:
I think this was changed by accident? The PR doesn't change the gui plugins path